README.md

```diff
@@ -55,7 +55,7 @@ The `DxDataChartCard` component displays custom metrics from DX datafeed endpoin
 
 - `title`: Card title
 - `description`: Optional card description/subtitle
-- `datafeedToken`: Token for the DX datafeed endpoint
+- `datafeedToken`: Token for the DX datafeed endpoint - see [datafeed docs](https://docs.getdx.com/data-studio/#datafeeds)
```

Comment:
Small bonus change - I wasn't familiar with the process of sharing a datafeed when reviewing some code, so this link helped me.